packages/scripts/src/commands/app-data/convert/processors/flowParser/parsers/template.parser.ts

```diff
@@ -92,6 +84,27 @@ export class TemplateParser extends DefaultParser {
     return flowsWithOverrides;
   }
 
+  /** Apply custom value transformations to rows with specific names, e.g. _list or _collection */
+  private transformRowValue(rowName: string, rowValue: any) {
+    if (rowName && rowValue && typeof rowValue === "string") {
+      // NOTE - required if passing an action_list from variable as only the `value`
+      // column is retained when interpreting data at runtime (workaround)
+      if (rowName.endsWith("_action_list") || rowName?.includes("_action_list_")) {
```

Comment:
I think we probably want to also allow for `rowName.startsWith("action_list_")`, e.g. `action_list_1` (or even allow for a var just called `action_list` too?)

Reply:
I'm keen to keep it consistent with the logic for `list` and `collection` keywords which have the exact same logic
```
 if (rowName.endsWith("_list") || rowName?.includes("_list_"))
 ....
 if (rowName.endsWith("_collection") || rowName.includes("_collection_")) {
 ```
 
 I have no issue changing all to allow the keyword to also be a prefix as well as suffix, but that would be a potentially breaking change so better done as a follow-up.

Reply:
yeh happy to leave as it is for now, I don't think it's too important especially as we're planning to move away from relying on these special value names eventually